test/lit.cfg

```diff
@@ -147,20 +148,27 @@ class SwiftTest(lit.formats.ShTest, object):
     def before_test(self, test, litConfig):
         if self.coverage_mode:
             profdir = self.profdir_for_test(test)
+            # FIXME: The compiler crashers run so fast they fill up the
+            # merger's queue (and therefore the build bot's disk)
+            if 'compiler_crashers' in profdir:
```

Comment:
Do you also want to cover `validation-test/SIL/crashers` and `validation-test/SIL/crashers_fixed`?